src/shared/bg_gameplay.h

```diff
@@ -35,6 +35,7 @@ along with Unvanquished Source Code.  If not, see <http://www.gnu.org/licenses/>
 
 #define BUILDER_DECONSTRUCT_RANGE 100
 
+constexpr int MAX_CREDITS = 2000;
```

Comment:
How about keeping the comment? You are the first one to complain about lack of doc :P
```suggestion
constexpr int MAX_CREDITS = 2000; // for aliens, CREDITS_PER_EVO converts this to evos for display
```